relayer/src/worker/packet.rs

```diff
@@ -26,6 +26,7 @@ pub struct PacketWorker<ChainA: ChainHandle, ChainB: ChainHandle> {
     chains: ChainHandlePair<ChainA, ChainB>,
     cmd_rx: Receiver<WorkerCmd>,
     clear_packets_interval: u64,
+    clear_packets_on_start: Option<bool>,
```

Comment:
Maybe we should define an `enum` to give more semantic meaning to the three possible states here?

eg.
```rust
enum OnStart {
  ClearPackets,
  DoNothing,
}

impl Default for OnStart {
  fn default() -> Self {
    Self::DoNothing
  }
}
```

and map

```rust
None => OnStart::default()
Some(true) => OnStart::ClearPackets
Some(false) => OnStart::DoNothing
```

Reply:
Nice idea! I'll add this. 👍